reckruitment/index.php

```diff
@@ -35,47 +37,66 @@
 
-<h4 class="mb-3">RRC/Athletics Bonding (Mandatory)</h4>
+<br>
+<h4 class="mb-3" id="gtky-events">RRC “Get To Know You” Events (Mandatory)</h4>
 
-<p>In order to provide applicants a casual setting to interact with Reck Clubbers and get a better feel for the kind of spirit Reck Club has for Georgia Tech, all applicants must attend one of the following GT Athletic events. Please note that applicants need only attend one of the following athletic events in order to satisfy the recruitment requirements:</p>
+    <p>In order to provide applicants a casual setting to interact with Reck Clubbers and get a better feel for the kind of spirit Reck Club has for Georgia Tech, all applicants must attend one of the following GT Athletic events. Please note that applicants need only attend one of the following athletic events in order to satisfy the recruitment requirements:</p>
 
 <ul>
-<!--    <li><strong>Sunday, January 20th</strong>: Women’s Basketball game vs. Syracuse at 2pm in McCamish Pavilion</li>-->
-<!--    <li><strong>Wednesday, January 23rd</strong>: Tech Rec Bowling at 7pm in the Student Center</li>-->
-<!--    <li><strong>Sunday, January 27th</strong>: Women’s Basketball game vs. NC State at 2pm in McCamish Pavilion</li>-->
+    <li><strong>Thursday, January 23rd</strong>: Women’s Basketball game vs. UNC at 6pm in McCamish Pavilion</li>
+    <li><strong>Wednesday, January 29th</strong>: Tech Rec Bowling Social from 7:30pm-9:30pm in the Student Center's Tech Rec</li>
+    <li><strong>Saturday, February 1st</strong>: Men’s Basketball game watch party vs. Notre Dame at 12:00pm (Location TBD)</li>
 <!--    <li><strong>Interviews: Tuesday, January 29th through Thursday, January 31st from 5:30-9pm (Mandatory)</strong></li>-->
-    <li>Check back here in December 2019 for athletic social dates for the 2020 recruitment cycle!</li>
+<!--    <li>Check back here in December 2019 for athletic social dates for the 2020 recruitment cycle!</li>-->
 </ul>
 
+    <br>
+<h4 class="mb-3" id="interviews">Interviews (Mandatory)</h4>
+
     <p>The interview portion of RECKruitment is designed to allow for Reck Clubbers to get to know applicants on a more personal basis. No need to stress! We just want to give each applicant the time to express their ideas, commitment, and personality. The interview stage will include a 20 minute individual interview followed by a short group interview activity. Dress code for this event is <strong>business semi-formal</strong>, and will be hosted in the Student Success Center Skyboxes. <a href="http://admission.gatech.edu/visit/bill-moore-student-success-center">Click here</a> for directions to the building, and be on the lookout for directions inside the building to the Skyboxes once you arrive. Interviewees will sign up for an interview time slot during check-in at the first social.</p>
```

Comment:
Let's also bold "Student Success Center Skyboxes" -- we want to pop out the most important information in each section